packages/dappmanager/test/utils/timeout.test.ts

```diff
@@ -0,0 +1,14 @@
+import "mocha";
+import { expect } from "chai";
+
+import { parseTimeout } from "../../src/utils/timeout";
+
+describe("Util: timeout", () => {
+  it("It should parse string parameter to number", () => {
+    const str = "20min"; // 12000 segs, 1200000ms
+    expect(parseTimeout(str)).to.equal(1200000);
+  });
+  it("It should return undefined timeout when the given parameter is undefined", () => {
+    expect(parseTimeout(undefined)).to.equal(undefined);
+  });
```

Comment:
Add a test for the number case

Reply:
Done. Anyway, I though that since the function only accepted `string | undefined `parameters, the function would not be never called with number as argument

Reply:
Right but when dealing with external uncontrolled inputs it's best to cover all cases